bin/incr

```diff
@@ -22,7 +25,7 @@ end
 desc('Increment the version of your NPM package.')
 command :npm do |cmd|
   cmd.action do |global_options, options, args|
-    npm = Incr::Command::Npm.new(args)
+    npm = Incr::Command::Npm.new(args, global_options)
```

Comment:
`global_options` is not passed to the constructor of the  `Mix` class, yet you are referencing it in the `initialize` method of said class.

Reply:
Fixed:
https://github.com/jcouture/incr/pull/2/files#diff-38802889bf3a5ca57d4a81f4d5643066R37